addons/product/migrations/13.0.1.2/pre-migration.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2020 ForgeFLow <http://www.forgeflow.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+_xmlid_renames = [
+    ('sale.group_discount_per_so_line', 'product.group_discount_per_so_line'),
+]
+
+
+def rename_image_attachments(env):
+    for old, new in [("image_variant", "image_variant_1024")]:
+        openupgrade.logged_query(
+            env.cr,
+            """
+            UPDATE ir_attachment SET res_field = %s
+            WHERE res_field = %s AND res_model = 'product.product'""",
+            (new, old),
+        )
+    for old, new in [("image", "image_1024"), ("image_medium", "image_128")]:
```

Comment:
@kos94ok-3D do you think we should this through ORM?

Reply:
Yes, I do.
And for product.product model too, because in v13 fields are stored.
In v12
![image](https://user-images.githubusercontent.com/33932458/77922317-913db000-72a9-11ea-95c7-88fb3bfa985c.png)
In v13
![image](https://user-images.githubusercontent.com/33932458/77922461-bdf1c780-72a9-11ea-899b-ce0d86c827b9.png)
@pedrobaeza What do you think about it? Do you found another solution?

Reply:
well, if I didn't misunderstand from what we did in `base`, the point of doing the rename by orm is that then the missing image fields are recomputed automatically by resizing the existing ones, right?

Reply:
Yes, we need to reprocess them after checking with a demo migration